rust/operator-binary/src/controller.rs

```diff
@@ -625,6 +625,11 @@ fn build_metastore_rolegroup_config_map(
                         Some(s3.endpoint().context(ConfigureS3Snafu)?.to_string()),
                     );
 
+                    data.insert(
+                        MetaStoreConfig::S3_REGION_NAME.to_string(),
+                        s3.region.name().map(String::from),
+                    );
+
```

Comment:
Note to self, we should only set this if `Some`.

Reply:
This now always sets `Some()` since 74934f2.